src/fender_value.rs

```diff
@@ -99,6 +158,59 @@ impl FenderValue {
             v => v.clone(),
         }
     }
+
+    pub fn remove_at(&mut self, pos: i64) -> Result<FenderReference, String> {
```

Comment:
Implement for string

Reply:
what would that return?

Reply:
what would that do on string?
(this msg may have been desynced for me)

Reply:
Remove a single character from the given index

Reply:
in vscode this comment showed up on a different function